app/models/cms/builtin.rb

```diff
@@ -111,10 +111,12 @@ class StaticPage < CMS::Builtin
       record.errors.add attr, :is_present if value.present?
     end
 
+    has_data_tag :builtin_page
+
     def to_xml(options = {})
       xml = options[:builder] || Nokogiri::XML::Builder.new
 
-      xml.builtin_page do |x|
+      xml.__send__(self.class.data_tag) do |x|
```

Comment:
Why this change for all classes? What is its benefit?

Not that I am opposing to it, but it makes it more difficult to understand what's going on (you need to go check what the `CMS::DataTag` module does... ).

Reply:
Maybe it's over-engineering. This came from `CMS::Section` and `CMS::Builtin::Section` sharing the same tag `<section>`. That was wrong and I needed to add a different tag for `CMS::Builtin::Section`, but to do so I would've needed to add a new `#to_xml` method to that class in order to call it's father only to change the root tag name, like in [CMS::Builtin::Partial](https://github.com/3scale/porta/blob/769f3623c35554933512dbdbc1b532d4aa70efac/app/models/cms/builtin.rb#L254-L256) and [CMS::Partial](https://github.com/3scale/porta/blob/769f3623c35554933512dbdbc1b532d4aa70efac/app/models/cms/partial.rb#L22). I thought that was too complicated and it would be good to simplify the models. I moved root tag names to a different module `CMS::DataTag` and unified all classes to use it. Thanks to that I could remove `#to_xml` also from `CMS::Builtin::Partial`.

Reply:
OK, thanks for the explanation!